### PR TITLE
Decay Learning Rate

### PR DESCRIPTION
**Pull request by jq**
learning rate decay each iteration 
learning_rate = max(learning_rate_decay*learning_rate, min_learning_rate)

@deerzq @saurfang

### CODE REVIEW

**Comment by deerzq** [comment]
LGTM
